lib/individual/spend_time.dart

```diff
@@ -0,0 +1,489 @@
+                    height: MediaQuery.of(context).size.height * 0.01,
+                  ),
+
+                  Padding(
+                    padding: EdgeInsets.only(
+                        left: MediaQuery.of(context).size.height * 0.023),
+                    child: Container(
+                      height: MediaQuery.of(context).size.height * 0.033,
+                      width: MediaQuery.of(context).size.height * 0.22,
+                      alignment: Alignment.topLeft,
+                      /* padding:  EdgeInsets.fromLTRB(MediaQuery.of(context).size.height * 0.023,
+                           MediaQuery.of(context).size.height * 0.025, MediaQuery.of(context).size.height * 0.138, MediaQuery.of(context).size.height * 0.38),*/
+                      child: Text(
+                        'Time',
+                        style: TextStyle(
+                          color: Colors.black,
+                          fontWeight: FontWeight.w700,
+                          fontFamily: 'PT Sans',
+                        ),
+                      ),
+                    ),
+                  ),
+
+                  SizedBox(height: MediaQuery.of(context).size.height * 0.01),
+                  Padding(
+                    padding: EdgeInsets.only(
+                        left: MediaQuery.of(context).size.height * 0.023),
+                    child: SizedBox(
+                      width: MediaQuery.of(context).size.height * 0.2,
+                      child: TextField(
+                        controller:
+                            timeinput, //editing controller of this TextField
+                        decoration: InputDecoration(
+                          border: OutlineInputBorder(
+                              borderRadius: BorderRadius.circular(
+                                  MediaQuery.of(context).size.height * 0.02)),
+                          // ignore: prefer_const_constructors
+                          fillColor: const Color.fromARGB(255, 186, 231, 235),
+                          filled: true,
+                          suffixIcon:
+                              Icon(Icons.arrow_drop_down), //icon of text field
+                          labelText:
+                              " Time", //labelStyle: TextStyle(fontSize:  MediaQuery.of(context).size.height*0.036)
+                          //label text of field
+                        ),
+
+                        readOnly:
+                            true, //set it true, so that user will not able to edit text
+                        onTap: () async {
+                          TimeOfDay? pickedTime = await showTimePicker(
+                            initialTime: TimeOfDay.now(),
+                            context: context,
+                          );
+
+                          if (pickedTime != null) {
+                            print(pickedTime.format(context)); //output 10:51 PM
+                            DateTime parsedTime = DateFormat.jm()
```

Comment:
An error is being encountered here whenever the user selects the time. Try to fix it please. Error details are provided in the attachment with this comment.
![Screenshot_20220904_083453](https://user-images.githubusercontent.com/106133014/188295249-300038ce-34e0-4c33-a561-99b2948da207.png)